src/Projections/Reunion/Reunion.csproj

```diff
@@ -0,0 +1,53 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
we shouldn't add a new projection - just update the existing WinUI projection to use Reunion